src/Oracles/Config/Setting.hs

```diff
@@ -51,6 +52,13 @@ data Setting = BuildArch
              | GmpLibDir
              | IconvIncludeDir
              | IconvLibDir
+             | Prefix
+             | BinDir
+             | LibDir
+             | Install
+             | InstallData
+             | InstallProgram
+             | InstallDir
```

Comment:
I don't quite agree with this: first I don't think it was in alphabetic order previously (sorry for being a bit mean), which hints its unnecessary maintenance burden, and second I think it makes more sense to group entries together by categories (prefix is certainly also one way to categorize things)